### PR TITLE
Phase II.G: Shared SEA Minter

### PR DESCRIPTION
**Pull request by ryley-o**
## Description of the change

Add handlers for ISharedMinterSEAV0.

## Tests
- e2e tests are added for ll events in `ISharedMinterSEAV0.sol`
- (stated for clarity) e2e tests are not added for events in other interfaces already tested, even if they are emitted by `MinterSEAV1`.

### CODE REVIEW

**Comment by ryley-o** [comment]
gentle reminder here @jakerockland @yoshiwarab

**Comment by jakerockland** [comment]
> gentle reminder here @jakerockland @yoshiwarab

Thank you for the ping ser – TAL today!

**Review by jakerockland — APPROVED** [review]
TY ser 🙏

File: src/i-shared-sea-mapping.ts [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+  );
+  setProjectMinterConfigExtraMinterDetailsValue(
+    "minBidIncrementPercentage",
+    BigInt.fromI32(event.params.minBidIncrementPercentage),
+    projectMinterConfig
+  );
+  projectMinterConfig.save();
+
+  // induce sync if the project minter configuration is the active one
+  updateProjectIfMinterConfigIsActive(
```
Comment (yoshiwarab) [inline src/i-shared-sea-mapping.ts:132]: Definitely not something we need to change now, but we decided we want to start treating ProjectMinterConfigs as first class sync entities right?

**Review by yoshiwarab — APPROVED** [review]

File: src/i-shared-sea-mapping.ts (hunk 1 above)
Comment (ryley-o) [reply src/i-shared-sea-mapping.ts:132]: yup! There is an Asana polish task for that, and it specifically calls out some extra helpful benefits on the SEA minter 🙏  https://app.asana.com/0/1203634050572702/1205060294151483/f

**Pull request merged**